ImpeccableHQ/swapr-dapp
# fix: remove market price bug on currency change

# Summary

Remove market price bug on currency change.

Previously the limit price didn't match the input labels and market price difference when changing currency.

  # To Test

1. Go to limit orders
2. Change to a new currency
3. Immediately swap limit order type (from sell to buy or vice-versa)   
- [ ] Everything should be consistent on the UI
